### ISSUE TITLE
Building in Dockerfile works but needs environment variable

### ISSUE THREAD

**Issue by rpgoldman** [ths-rwth/smtrat#110]
I was able to get my [Dockerfile](https://github.com/rpgoldman/smtrat/blob/debug-build/Dockerfile) to build SMTRAT successfully, but only after I set the environment variable `CARL_REQUIRED_VERSION`.  Should that be set to be the same as the version of SMTRAT somehow by the cmake files?

**Comment by derjasper** [comment]
Did you try deleting the CMakeCache.txt?

**Comment by derjasper** [comment]
There is nothing wrong about setting CARL_REQUIRED_VERSION, however, it should be set automatically.

**Comment by rpgoldman** [comment]
Yes, Docker does do caching, but my Dockerfile is quite rudimentary. I don't think the caching can apply.

I am also concerned about the fact that I needed to set that version.

Are you at all interested in having a Dockerfile for SMTRAT? It would have the advantages that

1. You could have GitHub actions run it after each release here as a way of testing that the build works properly outside your home environment.
2. You could push the docker containers to DockerHub and not have to worry about people like me complaining about their build problems!
